media-api/app/MediaApiComponents.scala

```diff
@@ -72,7 +72,7 @@ class MediaApiComponents(context: Context) extends GridComponents(context) {
 
   val imageResponse = new ImageResponse(config, s3Client, usageQuota)
 
-  val mediaApi = new MediaApi(auth, messageSender, elasticSearch, imageResponse, config, controllerComponents, s3Client, mediaApiMetrics)
+  val mediaApi = new MediaApi(auth, messageSender, elasticSearch, imageResponse, usageQuota.usageStore, config, controllerComponents, s3Client, mediaApiMetrics)
```

Comment:
Would be nice to extract the usageStore to a top level component and then insert just that, but not to fussed about it.